docs/windows.md

```diff
@@ -0,0 +1,187 @@
+  * The kernel driver of OVS should be [signed by Windows Hardware Dev Center](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/driver-signing)
+  * If OVS driver is not signed, please refer to the Windows doc about how to [install a test-signed driver ackage on the test computer](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/installing-a-test-signed-driver-package-on-the-test-computer).
+  *  If you don't have a self-signed OVS package and just want to try the Antrea on windows, Antrea provides a test-signed OVS package for you. See details in [Join Windows worker nodes](#Join-Windows-worker-nodes) section.
+
+### Installation
+#### Download & configure Antrea for Linux
+Configure the Antrea for Linux on master node following [Getting started](/docs/getting-started.md) document.
+```
+# Example:
+kubectl apply -f  https://raw.githubusercontent.com/ruicao93/antrea/windows_install/build/yamls/antrea-windows.yml
+```
+#### Add Windows Antrea Agent and kube-proxy DaemonSets
+Now you can add Windows-compatible versions of Antrea Agent and kube-proxy by applying yaml files privided by Antrea:
+- antrea-windows.yml
+  - Deploy Antrea Agent windows DaemonSet.
+- kube-proxy-windows.yaml:
+  - Deploy kube-proxy windows DaemonSet.
+
+Download and apply `antrea-windows.yml`.
+```
+# Example:
+kubectl apply -f https://github.com/ruicao93/antrea/raw/windows_install/build/yamls/antrea-windows.yml
+```
+Download `kube-proxy-windows.yaml` from kubernetes official repository.
+```
+curl.exe -LO https://github.com/ruicao93/antrea/raw/windows_install/build/yamls/kube-proxy-windows.yaml
+```
+Replace the content of run-script.ps1 in configmap named kube-proxy-windows as following:
+```
+apiVersion: v1
+data:
+  run-script.ps1: |-
+    $ErrorActionPreference = "Stop";
+    mkdir -force /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force /host/k/kube-proxy
+
+    cp -force /k/kube-proxy/* /host/k/kube-proxy
+    cp -force /var/lib/kube-proxy/* /host/var/lib/kube-proxy
+    cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
+
+    wins cli process run --path /k/kube-proxy/kube-proxy.exe --args "--v=4 --config=/var/lib/kube-proxy/config.conf --proxy-mode=userspace --hostname-override=$env:NODE_NAME"
+
+kind: ConfigMap
+metadata:
+  labels:
+    app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+``` 
+The apply the `kube-proxy-windows.yaml`
+```
+kubectl apply -f kube-proxy-windows.yaml
+```
+
+#### Join Windows worker nodes
+1.(Optional, Test-Only) Install OVS provided by Antrea
+
+Antrea provides a pre-build OVS package which contains test-signed OVS kernel driver. If you don't have a self-signed OVS package and just want to try the Antrea on windows, this package can be used for test. We also provide a script help to install the OVS driver and register userworld binaries as services.
```

Comment:
is "userworld" a common term for windows, or is it "userland" / "userspace"?

Reply:
Thanks Antonin, I think `userspace` is better.